.github/ISSUE_TEMPLATE/release-checklist.md

```diff
@@ -3,7 +3,9 @@ Release checklist:
 Tagging:
  - [ ] Write release notes in NEWS. Get them reviewed and merged
      - [ ] If doing a branched release, also include a PR to merge the NEWS changes into master
- - [ ] Ensure your local copy is up to date with master and your working directory is clean
+ - [ ] Ensure your local copy is up to date with master and should point to the upstream git repository i.e. `git@github.com:coreos/ignition.git`.
+ - [ ] Ensure your working directory is clean
+     - [ ] git clean -fd
```

Comment:
should probably be `git clean -fdx` to ensure that we're in a completely pristine state

Reply:
Also, this isn't a separate step from the previous line, so let's not ask the operator to check an additional box here.